CriticalMapsUITests/CriticalMapsUITests.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  CriticalMapsUITests.swift
+//  CriticalMapsUITests
+//
+//  Created by Felizia Bernutz on 06.10.19.
+//  Copyright © 2019 Pokus Labs. All rights reserved.
+//
+
+import XCTest
+
+class CriticalMapsUITests: XCTestCase {
```

Comment:
I'm wondering if we should run that test for one language and one device on the CI. 

We had a bug in the past were the app wasn't operable anymore with VoiceOver and Switch control and this test would have caught that.

Reply:
Yeah, sounds like a good idea!  I'll add another fastlane action which can run on the CI 👍

Reply:
fyi: There are now two jobs running in parallel. 

![image](https://user-images.githubusercontent.com/26111180/66272031-e5b92300-e864-11e9-840a-9f5126699cfb.png)